src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -431,6 +431,22 @@ public static SymbolInfo GetSymbolInfo(this SemanticModel semanticModel, SelectO
         }
 
         /// <summary>
+        /// Gets the symbol information for the property of a sub-property pattern.
+        /// </summary>
+        public static SymbolInfo GetSymbolInfo(this SemanticModel semanticModel, SubPropertyPatternSyntax node, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I am not sure it is appropriate to support GetSymbolInfo for SubPropertyPatternSyntax. The syntax node doesn't represent a reference to a symbol, its child node does.